nmma/em/likelihood.py

```diff
@@ -128,7 +128,12 @@ def log_likelihood(self):
             usedIdx = np.where(np.isfinite(mag_app_filt))[0]
             sample_times_used = self.sample_times[usedIdx]
             mag_app_used = mag_app_filt[usedIdx]
-            t0 = self.parameters["timeshift"]
+            try:
+                t0 = self.parameters["timeshift"]
+            except KeyError:
+                print("Warning: the 'KNtimeshift' parameter is deprecated as of nmma 0.0.19, please update your prior to use 'timeshift' instead")
+                t0 = self.parameters["KNtimeshift"]
+                continue
```

Comment:
There should not be a continue here, right?

Reply:
ah yeah, not sure why I included that